modules/ossm-rn-known-issues.adoc

```diff
@@ -92,8 +92,65 @@ To check HTTPS or TCP traffic by using an ingress gateway, you must have an exis
 
-* https://issues.jboss.org/browse/MAISTRA-622[MAISTRA-622]
-In Maistra 0.12.0/TP12, permissive mode does not work. The user has the option to use Plain text mode or Mutual TLS mode, but not permissive.
+* https://issues.jboss.org/browse/MAISTRA-622[MAISTRA-622] In Maistra 0.12.0/TP12, permissive mode does not work. The user has the option to use Plain text mode or Mutual TLS mode, but not permissive.
+
+* https://issues.jboss.org/browse/MAISTRA-684[MAISTRA-684] The Jaeger version in the `istio-operator` does not match the shipped version of Jaeger.
++
+If you install {ProductName} {ProductVersion} without changing the Jaeger version to 1.13.1, this is what you see when you check the pods in `istio-system`:
++
+----
+$ oc get pods -n istio-system
+
+NAME                                      READY   STATUS             RESTARTS   AGE
+elasticsearch-0                           1/1     Running            0          16m
+grafana-694d54c786-m6dfc                  2/2     Running            0          18m
+istio-citadel-7658c96954-r8p46            1/1     Running            0          18m
+istio-egressgateway-d759556b8-hwc7n       1/1     Running            0          18m
+istio-galley-7cf565999f-b729t             1/1     Running            0          18m
+istio-ingressgateway-bc97545d5-5mpw4      1/1     Running            0          18m
+istio-pilot-dd7c67fb5-r8nbt               2/2     Running            0          29m
+istio-policy-b5df8c557-5xbbl              2/2     Running            0          18m
+istio-sidecar-injector-5bccb75987-xl6t6   1/1     Running            0          18m
+istio-telemetry-7f86b4f6d9-kgl5p          2/2     Running            0          30m
+jaeger-collector-85c597698c-54b2c         0/1     ImagePullBackOff   0          18m
+jaeger-query-59b877c9d9-92vmj             1/3     ImagePullBackOff   0          18m
+kiali-54ff784b57-8clh2                    1/1     Running            0          18m
+prometheus-7b89468cf6-pbnqh               2/2     Running            0          18m
+----
++
+Run this command to see `istio-system` events:
++
+----
+$ oc get events -n istio-system
+----
++
+You will see this error:
++
+----
+...
+8m50s       Warning   Failed                         pod/jaeger-query-59b877c9d9-92vmj              Failed to pull image "registry.redhat.io/distributed-tracing-tech-preview/jaeger-agent:1.12.0": rpc error: code = Unknown desc = Error reading manifest 1.12.0 in registry.redhat.io/distributed-tracing-tech-preview/jaeger-agent: error parsing HTTP 404 response body: invalid character 'F' looking for beginning of value: "File not found.\""
+...
+----
++
+To ensure you are running Jaeger 1.13.1:
```

Comment:
I want to call this information out better, either moving it up (but that doesn't seem right either) or including the word "Workaround" here.  Because this feels like a long scroll to get to the solution.

Reply:
- I reordered the list of issues from newest to oldest. 
- I added sub-heads with anchors to the Service Mesh and Kiali Known issues so they're easier to find/navigate to
- I reordered the assembly to place known issues above fixed issues to reduce the amount of scrolling between new features and issues.